integreat_cms/cms/views/events/event_list_view.py

```diff
@@ -109,7 +109,7 @@ def get(self, request, *args, **kwargs):
 
         if settings.DEEPL_ENABLED:
             deepl = DeepLApi()
-            DEEPL_AVAILABLE = deepl.check_availability(request, language_slug)
+            DEEPL_AVAILABLE = deepl.check_availability(request, region.default_language)
```

Comment:
```suggestion
            DEEPL_AVAILABLE = deepl.check_availability(request, language)
```
Why the default language here?